cdap-unit-test/src/main/java/co/cask/cdap/test/internal/DefaultApplicationManager.java

```diff
@@ -211,40 +184,7 @@ public void waitForFinish(long timeout, TimeUnit timeoutUnit) throws TimeoutExce
   public WorkflowManager startWorkflow(final String workflowName, Map<String, String> arguments) {
     final Id.Program workflowId = Id.Program.from(applicationId, ProgramType.WORKFLOW, workflowName);
```

Comment:
``startProgram``? Or perhaps not, because workflows are started via schedules? Not completely sure.